### ISSUE TITLE
Airlearning Compilation issue 

### ISSUE THREAD

**Issue by pdx97** [harvard-edge/airlearning-ue4#9]
i have followed all the steps as directed in your GitHub and even install all the prerequisites as directed but after moving the airsim plugin folder to the UE-4 folder and when I try to run the airlearning.uproject i get an error saying "**Airlearning could be compiled. Try Rebuilding from source manually** " . the error logs say this [
![LOGFILE](https://user-images.githubusercontent.com/28250686/141719789-6059aa6f-61e0-4043-a0b7-ff0cefd385b8.PNG)
](url)

**Comment by BazilaAfridi** [comment]
@pdx97 Hello sir, wanted to ask that were you or anybody else able to solve this error? Because currently I'm having this error :(

**Comment by DjAzDeck** [comment]
I am facing the same error even without AirSim plugin, just trying to compile Air Learning. Using Unreal 4.25. 
Have you find any solution?

**Comment by pdx97** [comment]
> I am facing the same error even without AirSim plugin, just trying to compile Air Learning. Using Unreal 4.25. Have you find any solution?

Hi @DjAzDeck i had some issue with 4.25 kindly try reinstalling with 4.18 and VS2014 with a cmake of 3.14

**Comment by DjAzDeck** [comment]
> > I am facing the same error even without AirSim plugin, just trying to compile Air Learning. Using Unreal 4.25. Have you find any solution?
> 
> Hi @DjAzDeck i had some issue with 4.25 kindly try reinstalling with 4.18 and VS2014 with a cmake of 3.14

Yeah, seems like the only viable solution. Still, it is funny that you may wanna use [AirSim](https://microsoft.github.io/AirSim/build_windows/) that requires UE >= 4.25 .

**Comment by charlo1998** [comment]
> VS201

does it work for you with these versions? I'm using Unreal 4.18 and VS2017 as written in the readme, and I'm able to build the environment simulator without air sim, but I'm not able to build the given version of airsim

**Comment by qinglong0276** [comment]
@DjAzDeck @charlo1998 你们是否解决上面的问题?
